packages/config/tailwind.config.js

```diff
@@ -242,7 +242,7 @@ const uiConfig = ui({
               backgroundColor: 'hsl(var(--background-surface-200))',
               border: '1px solid ' + 'hsl(var(--background-surface-300))',
               borderRadius: theme('borderRadius.lg'),
-              wordBreak: 'break-all',
+              // wordBreak: 'break-all',
```

Comment:
@saltcod @fsansalvadore 

possible break here. 
I'll merge it anyway as it's not critical, but just so y'all know.